autosubmit/platforms/paramiko_submitter.py

```diff
@@ -151,7 +151,6 @@ def load_platforms(self, asconf, retries=5, auth_password = None, local_auth_pas
                 else:
                     platform_type_value = platform_type or "<not defined>"
                     raise AutosubmitCritical(f"PLATFORMS.{section.upper()}.TYPE: {platform_type_value} for {section.upper()} is not supported", 7012)
-                remote_platform.main_process_id = os.getpid()
```

Comment:
:+1: looks like it had been removed and we missed this one. Thanks for fixing it!